plugins/textpattern.md

```diff
@@ -101,10 +101,10 @@ The block patterns do not have an `end` property. This allows for patterns to be
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugin: 'textpattern lists',
-  textpattern_patterns: [
-    {start: '#', format: 'h1'},
+  selector: 'textarea',          // => change this value according to your HTML
+  plugin: 'textpattern lists',  // => make sure to have the required plugins here,
```

Comment:
Is this why we added `=>` to try and do a multiline comment? If so, that's really confusing and the multi line comment should be above the `plugin:  'textpattern lists'` line.

Reply:
Done